lib/buildsystems/cmake.rb

```diff
@@ -5,16 +5,16 @@ class CMake < Package
 
   attr_accessor :cmake_build_relative_dir
 
-  def initialize
-    super
-    @cmake_build_relative_dir = '.'
-  end
-
   def self.build
-    puts "Additional cmake_options being used: #{@pre_cmake_options.nil? ? '<no pre_cmake_options>' : @pre_cmake_options} #{@cmake_options.nil? ? '<no cmake_options>' : @cmake_options}".orange
+    @cmake_build_relative_dir ||= '.'
     @crew_cmake_options = @no_lto ? CREW_CMAKE_FNO_LTO_OPTIONS : CREW_CMAKE_OPTIONS
+    puts "Additional cmake options being used:".orange
+    method_list = methods.grep(/cmake_/).delete_if { |i| send(i).blank? }
```

Comment:
this feels like a much more complicated solution than the previous method of a puts statement, and also looks like it doesn't provide as much information as before in cases where the option is nil?

Reply:
By nil you mean the option isn't set? Because in that case it just doesn't show the method.

Reply:
Whereas before it would print something like <no_cmake_options>, right?